src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -59,6 +60,14 @@ void GcsServer::Start() {
   // Init gcs pub sub instance.
   gcs_pub_sub_ = std::make_shared<gcs::GcsPubSub>(redis_client_);
 
+  if (config_.grpc_pubsub_enabled) {
+    // Init grpc based pubsub
+    // TODO(before merging): Make these constants configurable.
```

Comment:
Please clear this TODO :-p

Reply:
lol whoops

thanks

Reply:
Should remove the TODO comment now that it's been addressed.